content/components/direction-links/rationale/arrows.md

```diff
@@ -4,3 +4,5 @@ heading: Arrows icons
 ---
 
 Arrow icons are used for direction links, rather than chevrons or triangles, to better articulate their purpose and separate the direction links from other elements which use chevron icons such as [accordions](/components/accordion/) or [call to action links](/components/cta-link/).
+
+The arrow icon is inside a span with an aria-hidden attribute set to true, this is to ensure that the arrow is not read out by a screen reader because it is a visual cue and is unneccessary information for a screenreader user.
```

Comment:
I think this could rational could be under its own heading.

And we can talk about how it was implemented previously.

Similar to what we did here:

https://designsystem.gov.au/components/tags/rationale/#deprecated-definition-list